raco/language/myrialang.py

```diff
@@ -632,7 +639,7 @@ def num_tuples(self):
     def compileme(self, inputid):
         if len(self.hash_columns) == 1:
             pf = {
-                "type": "SingleFieldHash",
+                "type": "RawValue" if self.is_raw_val else "SingleFieldHash",
```

Comment:
Better name might be `IdentityHash` or something like that. I didn't know what RawValue meant by reading it